saraho1123/overlook-sosgood
# Feature/manager interactions

## Is this a fix or a feature?
feature

## What is the change?
Completes all functionality for manager interactions:
I should be able to search for any user by name and:
* View their name, a list of all of their bookings, and the total amount they’ve spent
* Add a room booking for that user
* Delete any upcoming room bookings for that user (they cannot delete a booking from the past)

## What does it fix?
Found bugs as I was working that I fixed. 

## Where should the reviewer start?
line 26 of Fetch.js
line 8 of Guest.js
line 21 of Hotel.js
line 2 of Manager.js
line 212 of base.scss
line 11 of index.html
line 29 of index.js

## How should this be tested?
with repo cloned:
run: npm start in terminal
open: http://localhost:8080/ in browser
login using either manager or customer(# 01-50) and password of 'Overlook2020'.
Using any numbers <01 or >50 should cause an error message to appear below the submit button.
Once correctly logged in, all interactions for a guest and manager should be working.
There also should (hopefully!) not be any major bugs!

## Relevant Tickets:
#18
and some for #17
